docs/go-logging-instrumentation.md

```diff
@@ -0,0 +1,78 @@
+---
+id: go-logging-instrumentation
+title: Scope Go Agent Logging Instrumentation
+sidebar_label: Logging Instrumentation
+---
+
+The Scope Go agent has support for capturing logs from this number of sources:
+
+- Test logs (`t.Log`), by capturing the internal log buffer of the test, includes support for source file and number, but not the exact timestamp of the log event.
+- Standard logger (`log.Println`, ...), by using a custom writer to save the log events with the exact timestamp, source and file number (using log flags).
+- Custom logger (`log.Logger` instance) is instrumented similar to the standard logger with the same features.
+- Standard output / error (`os.Stdout`, `os.Stderr`) by replacing those implementation with Pipes and consuming the output, includes support for the timestamp of the log event but not the source file.
```

Comment:
```suggestion
- Standard output/error (`os.Stdout` and `os.Stderr`) can be manually instrumented. Includes support for the timestamp of the log event but not the source file or line number information. It is not compatible with parallel tests.
```

Is the timestamp here exact? No, right?

Reply:
No, the timestamp is not exact, has the pipe write and read delay.

Reply:
I have updated the suggested change